glossary.yml

```diff
@@ -3952,9 +4246,9 @@
   en:
     term: "XML"
     def: >
-      A set of rules for defining HTML-like tags and using them to format documents
-      (typically data). XML achieved license plate popularity in the early 2000s,
-      but its complexity led many programmers to adopt [JSON](#json) instead.
+      A set of rules for defining HTML-like tags and using them to format
+      documents (typically data). XML was popular in the early 2000s, but its
```

Comment:
This answers https://github.com/carpentries/glosario/pull/95#discussion_r497675293, but will unfortunately clash with #95

Reply:
I made the same change to #95.